src/components/homePage/Faq.vue

```diff
@@ -1,19 +1,71 @@
 <template>
-<div id="faq">
-	<h2>Frequently Asked Questions</h2>
-	<!-- CREATE A DROPDOWN COMPONENT -->
-</div>
+  <div id="faq">
+    <h2>Frequently Asked Questions</h2>
+    <div class="questions" v-for="question in questions['questions']" :key="question">
+      <button v-on:click="show" class="accordion">{{question.question}}</button>
+      <div class="content">
+        <p>{{question.answer}}</p>
+      </div>
+    </div>
+  </div>
 </template>
 
 <script>
+import Questions from "../../assets/faq.json";
+
 export default {
-	name: "Faq",
-}
+  name: "Faq",
+  data() {
+    return {
+      questions: Questions
+    };
+  },
+  methods: {
+    show: function(event) {
+      // close any accordions that's open first
+      let accordions = document.getElementsByClassName("accordion");
+
+      for (let element of accordions) {
+        element.nextElementSibling.style.maxHeight = null;
+      }
+
+      let content = event.target.nextElementSibling;
+      if (content.style.maxHeight) {
+        // accordion is open, we need to close it
+        content.style.maxHeight = null;
+      } else {
+        //accordion is closed
+        content.style.maxHeight = content.scrollHeight + "px";
+      }
+    }
```

Comment:
This is a pretty pure-JS way to do this. That's okay, but maybe in the future we can clean it up a bit with Vue :)